### PR TITLE
[Flann] disable openmp

### PR DESCRIPTION
**Pull request by Neumann-A**

### CODE REVIEW

**Comment by JackBoosY** [comment]
Any reason to do that?

**Comment by Neumann-A** [comment]
> Any reason to do that?

requires linkage with `libomp`. I don't know why it doesn't break with cl but clang-cl complains about missing symbols.

**Review by JackBoosY — APPROVED** [review]

**Comment by Osyotr** [comment]
Maybe convert that into a feature? I can imagine someone being upset after updating vcpkg and finding out that their program just got slower.

**Comment by JavierMatosD** [comment]
> Maybe convert that into a feature? I can imagine someone being upset after updating vcpkg and finding out that their program just got slower.

I agree with @Osyotr. This should be a default feature, and you can disable it for your use case.

File: ports/flann/portfile.cmake [hunk 1]
```diff
@@ -26,6 +26,7 @@ vcpkg_cmake_configure(
         -DBUILD_DOC=OFF
         -DBUILD_PYTHON_BINDINGS=OFF
         -DBUILD_MATLAB_BINDINGS=OFF
+        -DUSE_OPENMP=OFF
```
Comment (JavierMatosD) [inline ports/flann/portfile.cmake:29]: This should be a feature.

**Review by JavierMatosD — CHANGES_REQUESTED** [review]

File: ports/flann/portfile.cmake (hunk 1 above)
Comment (JavierMatosD) [reply ports/flann/portfile.cmake:29]: Actually, never mind. This is an improvement over the existing optional detection. We would like to make it a feature in the future.

**Review by JavierMatosD — APPROVED** [review]

**Comment by JavierMatosD** [comment]
Thanks!

**Pull request merged**